### DISCUSSION TITLE
Fetch data from DB using a date range

### DISCUSSION THREAD

**Discussion by mahes-2** [supabase/supabase#20986] [Questions] [answered]
I want to use the PostgREST query to fetch all the records that were created today. The created_at column stores a UTC timestamp. What is the appropriate way to do it in React Native?

**Comment by GaryAustin1** [comment, marked as answer]
https://postgrest.org/en/stable/how-tos/working-with-postgresql-data-types.html#timestamps
Shows using Postgres naming for date/time info.

You can also use js timestamps like this `.lte('start_time', new Date().toISOstring())` with multiple filters.

**Reply by mahes-2** [reply]
Thanks, Gary. Passing the timestamp this way worked.
